tasks/purge.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+# Purge AWS 
+
+  - name: Delete ALB
+    elb_application_lb:
+      name: "{{ lc_asg.alb.name }}"
+      region: "{{ lc_asg.aws_region }}"
+      state: absent
+
+  - name: Delete target group
+    elb_target_group:
+      name: "{{ lc_asg.tg.name }}"     
+      region: "{{ lc_asg.aws_region }}"
+      state: absent
+
+  - name: Delete ASG with discovered tg arn
+    ec2_asg:
+      name: "{{ lc_asg.aws_asg_name }}"
+      launch_config_name: "{{ lc_asg.aws_lc_name }}"
+      region: "{{ lc_asg.aws_region }}"     
+      state: absent
+
+  - name: Delete launch configuration
+    ec2_lc:
+      name: "{{ lc_asg.aws_lc_name }}"
+      region: "{{ lc_asg.aws_region }}"     
+      instance_type: "{{ lc_asg.aws_instance_type }}"
+      state: absent
```

Comment:
Please add purge for ecs cluster as well

Reply:
what will happen if tg is not defined? the state will fail i think, same with alb, condition should be added:
```
when: lc_asg.tg/alb is defined
```